ledger/store/src/helpers/rocksdb/internal/mod.rs

```diff
@@ -122,12 +124,16 @@ impl Database for RocksDB {
     ///
     /// In production mode, the database opens directory `~/.aleo/storage/ledger-{network}`.
     /// In development mode, the database opens directory `/path/to/repo/.ledger-{network}-{id}`.
-    fn open<S: Clone + Into<StorageMode>>(network_id: u16, storage: S) -> Result<Self> {
-        static DB: OnceCell<RocksDB> = OnceCell::new();
+    /// In tests, the database opens an ephemeral directory in the OS temporary folder.
```

Comment:
Nit: can you add a comment for Custom for completeness

Reply:
Done; I'll include it with the next rebase or any other review-related changes.